### PR TITLE
First Draft at a PGSQL usage doc

### PR DESCRIPTION
**Pull request by benjaminwgordon**
First draft at generating a useful PGSQL usage doc for reference/onboarding purposes

### CODE REVIEW

File: content/support/pgsql-guide.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+- organizations
+- email configurations
+- access tokens
+- plus many more relational data
+
+## Connecting to the PGSQL Database
+
+### Kubernetes:
+
+- kubectl exec -ti PGSQL_CONTAINER_NAME -- psql -U sg
```
Comment (Kelvin-Lee) [inline content/support/pgsql-guide.md]: I'd probably have certain lines between Line 30 - 41 formatted to use the `code` markup, though I suppose this is mostly a stylistic preference.

Namely, Line 30, 34, 38, 40, and 41

**Review by alexAtSourcegraph — COMMENTED** [review]
This looks great. Info is concise and the labeling is good for a quick scan to find the information I'm looking for. Thanks for doing this.

File: content/support/pgsql-guide.md (hunk 1 above)
Comment (benjaminwgordon) [reply content/support/pgsql-guide.md]: Good call, I like this edit

File: content/support/pgsql-guide.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
+# Navigating the PGSQL Database
+
+## The PGSQL Database
```
Comment (Kelvin-Lee) [inline content/support/pgsql-guide.md]: I'd consider some consistency regarding [ PGSQL | pgsql ] [ database | Database ] throughout, where appropriate (Example: PGSL Databse? Or PGSQL database? Etc.).

I have no strong opinions on which is best.

**Review by Kelvin-Lee — COMMENTED** [review]
Noice work Ben! 🙌

File: content/support/pgsql-guide.md (hunk 2 above)
Comment (benjaminwgordon) [reply content/support/pgsql-guide.md]: Good catch, resolved

File: content/support/pgsql-guide.md (hunk 1 above)
Comment (benjaminwgordon) [reply content/support/pgsql-guide.md]: Resolved

**Comment by abeatrix** [comment]
LGTM! Would adding the link to our [database schema ](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/internal/database/schema.md )be helpful?

**Review by abeatrix — APPROVED** [review]

**Pull request merged**